packages/version-bump/src/main.ts

```diff
@@ -62,6 +69,22 @@ export async function run(): Promise<void> {
 				build++;
 				break;
 
+			case 'micro_zip':
+				// for micro zips we bump back the minor number and replace rc with p.
+				if (patch === 0) {
+					if (minor === 0) {
+						major--;
+						minor = 9;
+						patch = 9;
+					} else {
+						minor--;
+					}
+				} else {
+					patch--;
+				}
+				release = releaseTypes.release;
```

Comment:
`releaseTypes.release` reads a bit odd, some internals might be improved

Reply:
Yeah, it should be something like `releaseTypes.production`, but it's some thing that's already used by many addons, so I didn't change the name.